multi-skill/src/bin/mmr_from_cc.rs

```diff
@@ -0,0 +1,122 @@
+use multi_skill::data_processing::{get_dataset_by_name, read_csv, try_write_slice_to_file};
+use multi_skill::summary::make_leaderboard;
+use multi_skill::systems::{
+    simulate_contest, EloMMR, EloMMRVariant, Player, PlayerEvent, PlayersByName,
+};
+
+use serde::{Deserialize, Serialize};
+use std::cell::RefCell;
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+struct SimplePlayer {
+    handle: String,
+    cur_mu: f64,
+    cur_sigma: Option<f64>,
+    num_contests: Option<usize>,
+}
+
+fn make_checkpoint(players: Vec<SimplePlayer>) -> PlayersByName {
```

Comment:
Maybe include a comment that this struct & function are duplicated in another file, so I might remember to extract them someday.